src/styles/main.scss

```
@@ -1,3 +1,61 @@
+@import "./utils/variables";
+
 body {
   margin: 0;
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+
+  width: (($daySize + $gapSize) * 7);
```

Your calculation should include paddings and only 6 gaps 
```suggestion
  width: ($daySize * 7) + ($gapSize * 6) + 20px;
```
The `( )` are just for clarity you don't really need them